frontend/model/settings/vuexModule.js

```diff
@@ -70,19 +70,20 @@ const mutations = {
     state.increasedContrast = isChecked
   },
   setNotificationEnabled (state, enabled) {
-    if (state.notificationEnabled !== enabled && enabled) {
```

Comment:
By removing this check, as it originally was, `state.notificationEnabled !== enabled`, you're potentially introducing infinite loops, and also triggering logic that's unnecessary. If `notificationsEnabled` is true, it means that we're in the process of establishing a push subscription or have already got one. Calling `service-worker/setup-push-subscription` in either of these cases won't do anything (of value).

Reply:
It should not result in an infinite loop, that's why there's a condition `if (enabled)` around `setTimeout(() => sbp('state/vuex/commit', 'setNotificationEnabled', false), 100)`.